src/components/biggive-heading/biggive-heading.tsx

```diff
@@ -1,64 +1,42 @@
 import { Component, Prop, h } from '@stencil/core';
+import { brandColour } from '../../globals/brand-colour';
+import { headingTag } from '../../globals/heading-tag';
+import { spacingOption } from '../../globals/spacing-option';
 
 @Component({
   tag: 'biggive-heading',
   styleUrl: 'biggive-heading.scss',
   shadow: true,
 })
 export class BiggiveHeading {
-  /**
-   * Space above component
-   */
-  @Prop() spaceAbove: number = 0;
-  /**
-   * Space below component
-   */
-  @Prop() spaceBelow: number = 0;
+  @Prop() spaceAbove: spacingOption = 2;
 
-  /**
-   * Colour Scheme
-   */
-  @Prop() colour: string = 'primary';
+  @Prop() spaceBelow: spacingOption = 4;
 
-  /**
-   * HTML element
-   */
-  @Prop() htmlElement: string = 'h1';
+  @Prop() colour: brandColour = 'primary';
+
+  @Prop() htmlElement: headingTag = 'h1';
 
-  /**
-   * Heading size
-   */
   @Prop() size: number = 1;
 
-  /**
-   * Text alignment
-   */
   @Prop() align: string = 'left';
 
-  /**
-   * Heading text
-   */
   @Prop() text: string = '';
 
-  /**
-   * Icon
-   */
   @Prop() icon: boolean = false;
 
-  /**
-   * Icon colour
-   */
-  @Prop() iconColour: string = 'primary';
+  @Prop() iconColour: brandColour = 'primary';
```

Comment:
You've got the `brandColour` type defined as below - but don't we also need to allow the campaign specific colours here, e.g. wgmf-purple, gmf-green etc.

Since they have pretty much the same name I think we probably want either the `$brandColours` scss variable and the `brandColour` typescript type to have the same list of colours in them - or to have more distinct names.

```
  | 'primary'
  | 'secondary'
  | 'tertiary'
  | 'brand-1'
  | 'brand-2'
  | 'brand-3'
  | 'brand-4'
  | 'brand-5'
  | 'brand-6'
  | 'black'
  | 'white'
  | 'grey-light'
  | 'grey-dark';
```

Reply:
Sorry deleting that comment I see brand-3 is the same thing as $colour-brand-wgmf-purple etc etc.